pywbemtools/pywbemcli/_connection_repository.py

```diff
@@ -70,15 +136,18 @@ class ConnectionRepository(object):
     connection exists on disk between pywbemcli sessions and within an
     instance of ConnectionRepository while pywbemcli is running.
 
-    ConnectionRepository lazy load  gets the repository from the disk so that
-    the repository is not in memory until one of the access methods is
-    executed or a write is executed.  This also allows creation of new
-    connections files using the connection save command.
+    ConnectionRepository _load is a lazy load that when it is needed rather
```

Comment:
What is `_load` ? The method is named `_load_connections_file()`, but that is not lazy, only its calling is.
Also, grammar issues with the rest of the sentence.
Maybe change the sentence to: 
```
The connections file is loaded in a lazy manner, so that it is loaded only
when needed rather than when the ConnectionRepository object is created.
```